packages/devtools_app/lib/main.dart

```diff
@@ -6,9 +6,12 @@ import 'package:flutter/material.dart';
 
 import 'src/app.dart';
 import 'src/config_specific/framework_initialize/framework_initialize.dart';
+import 'src/config_specific/theme_overrides/theme_overrides.dart';
```

Comment:
maybe rename theme_overrides to ide_theme as that is exactly what it is.
It is the theme of the IDE that we are embedding devtools in / using devtools with?

Reply:
Yeah, makes sense - I was trying to make it a bit more generic, but I'm not sure it really was. Done!